tests/epyccel/recognised_functions/test_epyccel_pyc_math.py

```diff
@@ -24,6 +24,20 @@ def call_gcd(x : int, y : int):
 
 # -----------------------------------------------------------------------------
 
+def test_call_gcd_negative(language):
+    def call_gcd(x : int, y : int):
+        from math import gcd
+        return gcd(x, y)
+
+    f = epyccel(call_gcd, language=language)
+    x = randint(-1e9, 1e9)
+    y = randint(-1e9, 1e9)
+
+    assert f(x,y) == call_gcd(x, y)
```

Comment:
Maybe it would be better to systematically test all combinations of positive/negative?

```suggestion
    x = randint(0, 1e9)
    y = randint(0, 1e9)

    assert f(x,y) == call_gcd(x, y)
    assert f(-x,y) == call_gcd(-x, y)
    assert f(x,-y) == call_gcd(x, -y)
    assert f(-x,-y) == call_gcd(-x, -y)
```

Reply:
Done [10bbc8a](https://github.com/pyccel/pyccel/pull/1986/commits/10bbc8a043c602393aaca161cad00906e5d25d6b)